src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator {
+    private String color;
+    private double area;
+    private String figureType;
```

Comment:
You don't need it here.

If you change your `AreaCalculator` interface - you can retrieve the area from the `calculateArea()` method.

For the figure type - you can specify it directly in every method `draw()` (in your current case `toString()`).